### ISSUE TITLE
Not work on the new version of mathscinet site

### ISSUE THREAD

**Issue by vanabel** [jhpalmieri/bibweb#1]
I checked the code, It seems the site of mathscinet is quite old, could your please update it?

**Comment by jhpalmieri** [comment]
Unfortunately the AMS doesn't document MathSciNet and the URLs it uses. Their current version uses a new syntax, and while I can figure out the basic usage, I do not see how to get directly to the bibtex format for the citations. They claim that the old version is available at https://mathscinet.ams.org/mathscinet/2006/mathscinet, but I can't get that to work.

**Comment by jhpalmieri** [comment]
Okay, now that my version of perl has the relevant modules updated, this should be fixed by pull request #2. Please test it.

**Comment by vanabel** [comment]
Thanks a lot for the work. In fact, I suspect that I can't contact with you, cause it is quite old project.

But it is useful for my scientific writhing, and I also try to fix it, it is worked somehow (not perfect), I would like to share my experience 
1. the get new bib from the mrnumber like is something like:
```perl
#https://mathscinet.ams.org/mathscinet/api/publications/format?formats=bib&ids=4504019
$new_wget_url = "\'https://mathscinet.ams.org/mathscinet/api/publications/format?formats=bib&ids=$mr\'";
```
2. I use the mrlookup to do the first search
```perl
if ($e_math =~ /
  ^mathscinet.ams.org$
  /x) {
  print "Using \`$e_math\' for the MathSciNet search.\n"
}
else {
  print "Warning: Your choice of \`$e_math\' for the MathSciNet site
  is not one of the recommended choices.  Proceeding anyway...\n\n";
}

$redirect = '';
#if ($e_math eq 'www.ams.org') {edirect = 'Providence%2C+RI+USA'}
#if ($e_math eq 'ams.rice.edu') {$redirect = 'Houston%2C+TX+USA'}
#if ($e_math eq 'ams.mathematik.uni-bielefeld.de') {
#    $redirect = 'Bielefeld%2C+Germany'}
#if ($e_math eq 'ams.mpim-bonn.mpg.de') {$redirect = 'Bonn%2C+Germany'}
#if ($e_math eq 'ams.u-strasbg.fr') {$redirect = 'Strasbourg%2C+France'}
#if ($e_math eq 'ams.impa.br') {$redirect = 'Rio+de+Janeiro%2C+Brazil'}
```

```perl
  # construct URL
  $wget_url =
  "\'https://$e_math/mrlookup?" .
  "au=$author&" .
  "ti=$title&" .
  "jrnl=$journal&" .
  "year=$yr&" .
  "format=bibtex&Search=Search'";
  unless ($use_stdout) {
    &bib_print("", "$bibtex_long_comment \n");
  }
```
which is unfortunately not support search by mrnumber.

I hope the above information can be help to improve the script.

**Comment by vanabel** [comment]
> Okay, now that my version of perl has the relevant modules updated, this should be fixed by pull request #2. Please test it.

I have test it, it works as expected, thanks a lot.

**Comment by jhpalmieri** [comment]
Thank you for the comments. This is fixed by PR #2.
